cypress/support/step-definitions/drawer-steps.js

```diff
@@ -0,0 +1,29 @@
+import {
+  drawerToggle,
+  drawerSidebar,
+  drawerSidebarContent,
+} from '../../locators/drawer';
+
+When('I click on Drawer arrow', () => {
+  drawerToggle().first().click();
+});
+
+Then('sidebar should have class {word}', (className) => {
+  drawerSidebar().should('have.class', className);
+});
+
+Then('sidebar text is visible', () => {
+  drawerSidebarContent().find('li:nth-child(1)').should('have.text', 'link a').and('be.visible');
```

Comment:
add Enter before `.and` line
```suggestion
  drawerSidebarContent().find('li:nth-child(1)').should('have.text', 'link a')
    .and('be.visible');
```

Reply:
fixed 4d965d0